pygmt/src/xyz2grd.py

```diff
@@ -14,14 +14,18 @@
 
 @fmt_docstring
 @use_alias(
+    A="duplicate",
```

Comment:
GMT.jl uses ['multiple_nodes' for `-A`](https://www.generic-mapping-tools.org/GMT.jl/v0.31/#GMT.xyz2grd), should we follow?

Reply:
I prefer `duplicate` but I think having parameters named consistently across wrappers is more important